src/lib/airlock/airlock.js

```diff
@@ -6,7 +6,7 @@ import { serviceLogin, serviceLogout } from '../redux/services';
 Airtable.configure({
   apiKey: 'airlock',
   // this must be updated by the tester
-  endpointUrl: 'https://790c-64-30-75-107.ngrok.io',
+  endpointUrl: 'https://3372-64-30-75-107.ngrok.io',
```

Comment:
this was an accidental commit (example of what you shouldn't do my b), please replace the string here with `Config.ENDPOINT_URL`